lib/skate_web/router.ex

```diff
@@ -106,6 +106,7 @@ defmodule SkateWeb.Router do
     get "/settings", PageController, :index
     get "/detours", PageController, :index
     get "/minimal", PageController, :index
+    get "/minimal/*id", PageController, :index
```

Comment:
question(non-blocking): what does the `*` mean here? I'm used to something like `:id` but not `*id`, is this a way to get phoenix to ignore the parameter but know it's there?

Reply:
Ohh huh! I didn't realize I was using both in different situations without a lot of nuance. It is more accurate to use `:` here. The `*` allows the rest of the matching url to be collected into the `id` param in the form `%{"id" => ["list", "of", "separated", "params"]}`. [Doc](https://hexdocs.pm/phoenix/Phoenix.Router.html#module-routing)